Backend/routes/auth.js

```diff
@@ -126,7 +183,8 @@ router.post("/add", restrictToPresident, async (req, res) => {
     });
     const pors = req.body.pos_res;
 
-    const dbUri = `mongodb+srv://${username}:${password}@cosa-database.xypqv4j.mongodb.net/?retryWrites=true&w=majority`;
+    // using local db for testing, in production
+    const dbUri = `mongodb+srv://${process.env.DB_USERNAME}:${process.env.DB_PASSWORD}@cosa-database.xypqv4j.mongodb.net/?retryWrites=true&w=majority`;
```

Comment:
Store the complete url in .env
```suggestion
    const dbUri = process.env.MONGODB_URL;
```

Reply:
The db username and password has always been there differently this would cause us to change it everywhere. Even the .env.example contains username and password separately instead of the entire URI. I suggest changing these configurations once project is done

Reply:
You can keep DB_USERNAME and DB_PASSWORD  as it is and add them as a variable in the link, something like this (pls verify it once):
 ```MONGODB_URL = "mongodb+srv://${DB_USERNAME}:${DB_PASSWORD}@cosa-database.xypqv4j.mongodb.net/?retryWrites=true&w=majority"```

Reply:
That's the way it is now right?

Reply:
I meant you can add this link in .env like this, so that you dont need to remove `DB_USERNAME` and `DB_PASSWORD`

Reply:
Oh understood! But would still need everyone to change their .env.

For all the reviews, Can we not wait till the project is done. Changing env variables at the middle of development and testing seems unnecessary work for everyone.

But it's ok tell me and I will change it everywhere if it's important. No problem in that as well.

Reply:
I'd still recommend adding the urls to .env, changing envs frequently is very common while creating projects 😄

Reply:
Ok, will do, thanks for the tip!

Reply:
@DevilsAutumn I tried this and did some research. I don't think you can use the `.env` variables inside the env file as it's never executed like a program, please correct me if I'm wrong.

Reply:
my bad. You can leave it as it is then.